### PR TITLE
Lock10 URL updates

### PR DESCRIPTION
**Pull request by jeffreylees**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by jeffreylees** [comment]
Also updated the `index.md` to reflect `Auth0` availability when including Lock via CDN or Bower

File: articles/libraries/lock/index.md [hunk 1]
```diff
@@ -120,21 +120,13 @@ This is just one example of how **Lock 10** could work with a **Single Page Appl
...
-
-Then execute `npm i` to install dependencies (only once) and `npm example` from the root of this project.
-
-Finally, point your browser at `http://localhost:3000/` and play around.
-
-## Start using Lock!
+## Using auth0.js
 
-${lockSDK}
+There is no need to use `getClient()` any more - instead, you can just instantiate a new `Auth0` object.
```
Comment (gnandretta) [inline articles/libraries/lock/index.md]: - Using "need" suggests that `lock.getClient()` is still available but it is not.
- The reader probably doesn't know what is an `Auth0` object, what is auth0.js, or why does she needs to use them, specially if this is the first time is using lock.

File: articles/libraries/lock/index.md [hunk 2]
```diff
@@ -120,21 +120,13 @@ This is just one example of how **Lock 10** could work with a **Single Page Appl
...
 
-## Using auth0.js
+To get `Auth0` with Lock, if you used the CDN or bower for installation, you merely have to instantiate your object (see below). If you are retrieving Lock via npm, you will have to include `auth0.js` in your project's dependencies and import it.
```
Comment (gnandretta) [inline articles/libraries/lock/index.md]: - "if you include the script from our CDN or installed Lock from bower"
- "if you installed Lock from npm"
- The npm package name is `auth0-js`

File: articles/libraries/lock/index.md [hunk 3]
```diff
@@ -120,21 +120,13 @@ This is just one example of how **Lock 10** could work with a **Single Page Appl
...
 
-When using functionality from auth0.js, instead of using `getClient()`, for example, you can simply instantiate a new Auth0 object and use plain auth0.js.
+In either case, you then simply use `auth0.js` by instantiating a new Auth0 object, as below.
```
Comment (gnandretta) [inline articles/libraries/lock/index.md:135]: If you are using the npm modue you need to require it first:

``` js
var Auth0 = require('auth0-js');
var client = new Auth0({ ... });
```

File: articles/libraries/lock/index.md [hunk 4]
```diff
@@ -145,6 +137,18 @@ var auth0 = new Auth0({
 });
 ```
 
+## Examples
```
Comment (gnandretta) [inline articles/libraries/lock/index.md]: I didn't understand what example is this referring to.

**Comment by mpaktiti** [comment]
Hi @jeffreylees can you rebase this PR to resolve the conflicts? Thanks.

**Comment by jeffreylees** [comment]
@mpaktiti conflicts resolved

**Pull request merged**